WordPress/Classes/ViewRelated/Stats/JetpackSettingsViewController.m

```diff
@@ -95,6 +95,8 @@ - (BOOL)hidesBottomBarWhenPushed
 
 - (void)willAnimateRotationToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrientation duration:(NSTimeInterval)duration
 {
+    [super willAnimateRotationToInterfaceOrientation:toInterfaceOrientation duration:duration];
```

Comment:
Cool. No worries. I think there might not need to be much of a change to the layout code if `[self layoutControls];` was called from the transition coordinator's completion method. Something to consider later on.